src/utils/token_to_fiat_oracle.ts

```diff
@@ -0,0 +1,130 @@
+import type { FiatOracle } from './fiat_oracle';
+import type { CoinGeckoPriceRequestParams, CoinGeckoPriceResponseData, CoinGeckoResponse } from './coingecko_types';
+import { TokenToFiatPrice } from './token_fiat_price';
+import type { FiatID, TokenID } from './fiat_oracle_types';
+
+const pollingIntervalMilliseconds = 1000 * 60 * 15;
```

Comment:
Lets leave a comment of `// 15 minutes` here

Reply:
Added at 4fc1b43c3640b7a0cfc720e016133f59b0996ff4